### DISCUSSION TITLE
Overwrite Layout

### DISCUSSION THREAD

**Discussion by ghost** [atrocore/atrocore#672] [Q&A] [answered]
Hello, 

sorry for the beginner question but how can i overwrite the layout code without it being "reset" after every update? 

For example
/var/www/html/client/modules/pim/res/templates/product-attribute-value/record/**list-in-product.tpl**

I changed the font-weight for the attribute group name, but i have to apply my change everytime i update the system using the usual process (php composer.phar update)

Is there any way to tag a file as "customized" so it does not get changed?

Thank you very much

**Comment by atrolex** [comment, marked as answer]
If you want just to change the CSS do it in administration / customisation / user interface.

**Reply by ghost** [reply]
I do some small adjustments to the html / templates too, not just the CSS

**Reply by atrolex** [reply]
As everything is modular and version-based, you need to create your own module for that, otherwise your things will be overwritten.
You can start here:
https://help.atrocore.com/#/atrocore/developer-guide/creating-own-module
